sass/_vars.scss

```diff
@@ -1,113 +1,310 @@
+                    light: palette(lightgray, 8),
+                    dark: palette(darkgray, 1),
+                ),
+            ),
+        ),
+        shadow: (
+            default: (
+                light: palette(lightgray, 8),
+                dark: palette(darkgray, 7),
+            ),
+            inset: (
+                light: palette(lightgray, 9),
+                dark: palette(darkgray, 7),
+            ),
+            subtle: (
+                light: palette(lightgray, 5),
+                dark: palette(darkgray, 6),
+            ),
+            focus: (
+                default: palette(indigo, 5),
+            ),
+        ),
+        accent: (
+            card: (
+                light: palette(blue, 3),
+                dark: palette(blue, 4),
+            ),
+            note: (
+                light: palette(green, 5),
+                dark: palette(green, 4),
+            ),
+            link: (
+                light: palette(blue, 7),
+                dark: palette(blue, 2),
+            ),
+            danger: (
+                light: palette(red, 5),
+                dark: palette(red, 4),
+            ),
+        ),
+        flag: (
+            1: (
+                light: palette(red, 5),
+                dark: palette(red, 4),
+            ),
+            2: (
+                light: palette(orange, 4),
+                dark: palette(orange, 3),
+            ),
+            3: (
+                light: palette(green, 4),
+                dark: palette(green, 3),
+            ),
+            4: (
+                light: palette(blue, 5),
+                dark: palette(blue, 4),
+            ),
+            5: (
+                light: palette(fuchsia, 4),
+                dark: palette(fuchsia, 3),
+            ),
+            6: (
+                light: palette(teal, 4),
+                dark: palette(teal, 3),
+            ),
+            7: (
+                light: palette(purple, 5),
+                dark: palette(purple, 4),
+            ),
+        ),
+        state: (
+            new: (
+                light: palette(blue, 5),
+                dark: palette(blue, 3),
+            ),
+            learn: (
+                light: palette(red, 6),
+                dark: palette(red, 4),
+            ),
+            review: (
+                light: palette(green, 6),
+                dark: palette(green, 5),
+            ),
+            buried: (
+                light: palette(amber, 5),
+                dark: palette(amber, 8),
+            ),
+            suspended: (
+                light: palette(yellow, 4),
+                dark: palette(yellow, 1),
+            ),
+            marked: (
+                light: palette(indigo, 2),
+                dark: palette(purple, 5),
+            ),
+        ),
+        highlight: (
+            bg: (
+                light: palette(blue, 3),
+                dark: palette(blue, 4),
+            ),
+            fg: (
+                default: black,
```

Comment:
And this?

Reply:
This was an oversight. It should be "light". Thank you!